src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class Storage {
+    private String filePath;
+
+    /**
+     * A constructor to specify the path
+     * @param filePath the path to save the file and load the file from
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Updates the file
+     * @param str The strings to be written in the file
+     */
```

Comment:
Nitpicking here, but there should be a blank line between description and the param section.